src/pudl/metadata/enums.py

```diff
@@ -190,3 +190,28 @@
     "plant_match_ferc1",
 }
 """The plant parts in the EIA plant parts list."""
+
+TECH_DESCRIPTIONS_NRELATB: set[str] = {
+    "AEO",
+    "Biopower",
+    "CSP",
+    "Coal_FE",
+    "Coal_Retrofits",
+    "CommPV",
+    "Commercial Battery Storage",
+    "DistributedWind",
+    "Geothermal",
+    "Hydropower",
+    "LandbasedWind",
+    "NaturalGas_FE",
+    "NaturalGas_Retrofits",
+    "Nuclear",
+    "OffShoreWind",
+    "Pumped Storage Hydropower",
+    "ResPV",
+    "Residential Battery Storage",
+    "Utility-Scale Battery Storage",
+    "Utility-Scale PV-Plus-Battery",
+    "UtilityPV",
+}
+"""NREL ATB technology descriptions."""
```

Comment:
I considered doing either encoding or otherwise renaming of these guys. I don't love that they are a mix of formats.